src/content/ja/2022/page-weight.md

```diff
@@ -0,0 +1,446 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: ページの重さ
+#TODO - Review and update chapter description
+description: 2022年のWeb Almanacのページの重さの章は、ページの重さが重要である理由、帯域幅、複雑なページ、時間と共に変化するページの重さ、ページリクエスト、ファイル形式について説明しています。
+authors: [fellowhuman1101, dwsmart]
+reviewers: [CSteele-gh]
+analysts: [drohe]
+editors: []
```

Comment:
Could you add me as an editor (also in other langs if you don't mind) as I did edit this in #3159